### PR TITLE
invoice doc correction

### PR DESCRIPTION
**Pull request by ankitdas13**
[Create an invoice (sdk audit)](https://docs.google.com/spreadsheets/d/1FwFHxOY994oS6jCmPd4ubdC4Itbr5QGaRg24_O6xj4U/edit#gid=0&range=A59) : [update](https://github.com/razorpay/razorpay-go/blob/invoice_doc_correction/documents/invoice.md?plain=1#L59)

[Update an invoice (sdk audit)](https://docs.google.com/spreadsheets/d/1FwFHxOY994oS6jCmPd4ubdC4Itbr5QGaRg24_O6xj4U/edit#gid=0&range=A60) : [update](https://github.com/razorpay/razorpay-go/blob/invoice_doc_correction/documents/invoice.md?plain=1#L266)

[Delete an invoice (sdk audit)](https://docs.google.com/spreadsheets/d/1FwFHxOY994oS6jCmPd4ubdC4Itbr5QGaRg24_O6xj4U/edit#gid=0&range=A62) :  You can only delete an invoice that is in the `draft` state

### CODE REVIEW

File: documents/invoice.md [hunk 1]
```diff
@@ -239,7 +253,7 @@ data:= map[string]interface{}{
     "updated-key": "An updated note.",
   },
 }
-body, err := client.Invoice.Invoice.Update("<invoiceId>", data, nil)
+body, err := client.Invoice.Update("<invoiceId>", data, nil)
```
Comment (sofiya02) [inline documents/invoice.md:257]: returns NULL

**Review by sofiya02 — CHANGES_REQUESTED** [review]
Delete an Invoice return NULL and not []

File: documents/invoice.md (hunk 1 above)
Comment (ankitdas13) [reply documents/invoice.md:257]: > returns NULL

can you try with this once 

`	line_items := make(map[string]interface{})
	line_items["0"] = map[string]interface{}{
		  "id": "li_JpNz5IZZcAtRFU",
		  "name": "Book / English August - Updated name and quantity",
		  "quantity": 1,
		}
	
	data:= map[string]interface{}{
	  "line_items": line_items,
	  "notes": map[string]interface{}{
		"updated-key": "An updated note.",
	  },
	}`

	`body, err := client.Invoice.Update("inv_JpNz5HmNXYvvNL", data, nil)`

**Review by sofiya02 — APPROVED** [review]
LGTM

**Pull request merged**